crfs.go

```diff
@@ -899,16 +919,17 @@ func direntType(ent *stargz.TOCEntry) fuse.DirentType {
 // node is a CRFS node in the FUSE filesystem.
 // See https://godoc.org/bazil.org/fuse/fs#Node
 type node struct {
-	fs *FS
-	te *stargz.TOCEntry
-	sr *stargz.Reader
-	f  *os.File // non-nil if root & in debug mode
+	fs     *FS
+	te     *stargz.TOCEntry
+	sr     *stargz.Reader
+	f      *os.File // non-nil if root & in debug mode
+	opaque bool     // true if this node is an overlayfs opaque directory
 
 	mu sync.Mutex // guards child, below
 	// child maps from previously-looked up base names (like "foo.txt") to the *node
 	// that was previously returned. This prevents FUSE inode numbers from getting
 	// out of sync
-	child map[string]*node
+	child map[string]fspkg.Node
```

Comment:
Tiny nit: the comment here is now slightly misleading, since this could return the `*node` or a whiteout.

Reply:
Thank you for your review. I fixed it.